kmip/pie/objects.py

```diff
@@ -446,12 +473,13 @@ class PrivateKey(CryptographicObject):
         cryptographic_algorithm: The type of algorithm for the PrivateKey.
         cryptographic_length: The length in bits of the PrivateKey.
         value: The bytes of the PrivateKey.
+        format_type: The format of the key value.
         cryptographic_usage_masks: The list of usage mask flags for PrivateKey
             application.
         names: The list of string names of the PrivateKey.
     """
 
-    def __init__(self, algorithm, length, value, masks=None,
+    def __init__(self, algorithm, length, value, format_type, masks=None,
```

Comment:
There are a ton of public/private key format types that we've never used. Pretty much all of them would not be supported here, since the key value is a byte string; the key value for the various transparent key formats are different types of structs. I think those three are reasonable for now.